analyzer/queries/queries.go

```diff
@@ -601,10 +601,13 @@ var (
 
 	RuntimeEVMNFTInsert = `
     INSERT INTO chain.evm_nfts
-      (runtime, token_address, nft_id, last_want_download_round)
+      (runtime, token_address, nft_id, owner, num_transfers, last_want_download_round)
     VALUES
-      ($1, $2, $3, $4)
-    ON CONFLICT (runtime, token_address, nft_id) DO NOTHING`
+      ($1, $2, $3, $4, $5, $6)
+    ON CONFLICT (runtime, token_address, nft_id) DO UPDATE
+    SET
+      owner = COALESCE(excluded.owner, chain.evm_nfts.owner),
+      num_transfers = chain.evm_nfts.num_transfers + excluded.num_transfers`
```

Comment:
Optional: For upserts, I've recently been using an `old` alias like so, and liking it:
```sql
INSERT INTO chain.my_long_named_table AS old
...
ON CONFLICT DO UPDATE SET
  num_transfers = old.num_transfers + excluded.num_transfers
```

Reply:
k let me add that too